libkbfs/backpressure_disk_limiter.go

```diff
@@ -337,36 +355,54 @@ func (bdl *backpressureDiskLimiter) getSnapshotsForTest() (
 	byteSnapshot, fileSnapshot bdlSnapshot) {
 	bdl.lock.RLock()
 	defer bdl.lock.RUnlock()
-	return bdlSnapshot{bdl.byteTracker.used, bdl.byteTracker.free,
-			bdl.byteTracker.semaphoreMax,
-			bdl.byteTracker.semaphore.Count()},
-		bdlSnapshot{bdl.fileTracker.used, bdl.fileTracker.free,
-			bdl.fileTracker.semaphoreMax,
-			bdl.fileTracker.semaphore.Count()}
+	return bdlSnapshot{bdl.journalByteTracker.used, bdl.journalByteTracker.free,
+			bdl.journalByteTracker.semaphoreMax,
+			bdl.journalByteTracker.semaphore.Count()},
+		bdlSnapshot{bdl.journalFileTracker.used, bdl.journalFileTracker.free,
+			bdl.journalFileTracker.semaphoreMax,
+			bdl.journalFileTracker.semaphore.Count()}
 }
 
 func (bdl *backpressureDiskLimiter) onJournalEnable(
 	ctx context.Context, journalBytes, journalFiles int64) (
 	availableBytes, availableFiles int64) {
 	bdl.lock.Lock()
 	defer bdl.lock.Unlock()
-	availableBytes = bdl.byteTracker.onJournalEnable(journalBytes)
-	availableFiles = bdl.fileTracker.onJournalEnable(journalFiles)
+	availableBytes = bdl.journalByteTracker.onEnable(journalBytes)
+	availableFiles = bdl.journalFileTracker.onEnable(journalFiles)
+	bdl.updateFreeLocked()
```

Comment:
this `updateFreeLocked` isn't strictly necessary; the next time either the journal or disk cache do a respective `Put`, they will call it again, correctly, and handle errors correctly.